service/plugins/perf/static/js/trend_chart.js

```diff
@@ -58,14 +91,40 @@ var drawSingleTrendChart = function(
       height: cfg.container.height,
     })
 
-  var ops = _.pluck(series, 'ops_per_sec');
+  var ops = _.pluck(series, cfg.valueAttr);
   var opsValues = _.pluck(series, 'ops_per_sec_values');
   var avgOpsPerSec = d3.mean(ops)
 
-  var hasValues = !_.contains(opsValues, undefined)
+  var allLevels = _.keys(series[0].threadResults)
+  var levels = threadMode == MAXONLY
+    ? [''+_.max(allLevels, function(d) { return +d })]
```

Comment:
Is it necessary to convert the value to a string here? I was pretty confused when I first read this